storagemarket/impl/provider.go

```diff
@@ -342,6 +347,10 @@ func (p *Provider) ImportDataForDeal(ctx context.Context, propCid cid.Cid, data
 		return xerrors.Errorf("given data does not match expected commP (got: %s, expected %s)", pieceCid, d.Proposal.PieceCID)
 	}
 
+	log.Debugw("will fire ProviderEventVerifiedData for imported file", "propCid", propCid)
+	if err := tempfi.Close(); err != nil {
+		log.Errorw("failed to close temp imported local file", "propCid", propCid, "err", err)
+	}
 	return p.deals.Send(propCid, storagemarket.ProviderEventVerifiedData, tempfi.Path(), filestore.Path(""))
```

Comment:
What is this empty filestore path? Is this expected? (I know this PR hasn't touched this)

Reply:
Hey , that's the "metdata -path", a remanent of the universal retrieval era.